lldb/unittests/Disassembler/CMakeLists.txt

```diff
@@ -1,11 +1,32 @@
+set(disas_srcs "")
+
 if("ARM" IN_LIST LLVM_TARGETS_TO_BUILD)
-  add_subdirectory(ARM)
+  set(disas_srcs ${disas_srcs}
```

Comment:
Suggestion: use `list(APPEND disas_srcs ...)` to more clearly express the intent that you're collecting a list of source files.

This one would be:
```
list(APPEND disas_srcs
    ARM/TestArm64Disassembly.cpp
    ARM/TestArmv7Disassembly.cpp
)
```

Reply:
Good idea, done.